recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "coffea" %}
-{% set version = "2024.11.0" %}
+{% set version = "2025.1.0" %}
 {% set python_min = "3.9" %}
```

Comment:
Already picked up the `3.9` that would have been needed from https://github.com/scikit-hep/coffea/pull/1233 from dependencies forcing newer releases.